content/docs/hello-world.md

```diff
@@ -15,36 +15,43 @@ ReactDOM.render(
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Het weergeeft een kop met de tekst "Hello, world!" op de pagina.
 
-[](codepen://hello-world)
+[Probeer het op CodePen](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+Klik op de bovenstaande link om een online editor te openen.
+Voel je vrij om wat aanpassingen te maken en te zien hoe deze de uitvoer beïnvloeden.
+De meeste pagina's in deze gids hebben dit soort bewerkbare voorbeelden.
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## Hoe Lees Je Deze Gids {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+In deze gids zullen we de bouwstenen van React-applicaties onderzoeken: elementen en componenten.
+Van zodra je ze onder de knie hebt, kan je complexe applicaties ontwikkelen uit kleine, herbruikbare stukjes.
```

Comment:
```suggestion
Als je deze onder de knie hebt, kan je complexe applicaties ontwikkelen uit kleine, herbruikbare stukjes.
```

"Van zodra" is not something a Dutchman would use as an expression. It would either be "zodra", or more likely the simpler "als".

Reply:
I agree with both of you here. I will opt for "wanneer".